go/Makefile

```diff
@@ -0,0 +1,5 @@
+.asdf_ok: .tool-versions .envrc
+	./script/asdf_bootstrap.sh && touch .asdf_ok
```

Comment:
Probably could have the script do the touch, but since it's essentially there as a make dependency, I preferred it to be visible.